.github/workflows/manual-deploy-obscuro-gateway.yml

```diff
@@ -207,9 +207,11 @@ jobs:
             && docker network create --driver bridge node_network || true \
             && cd /home/obscuro/go-obscuro/ \
             && docker run -d -p 80:80 -p 81:81 --name "${{ env.VM_NAME }}" \
+              --entrypoint "/home/ten/go-ten/tools/walletextension/main/entry.sh" \
               -e OBSCURO_GATEWAY_VERSION="${{ GITHUB.RUN_NUMBER }}-${{ GITHUB.SHA }}" \
-               --log-opt max-file=3 --log-opt max-size=10m \
-               ${{ env.DOCKER_BUILD_TAG_GATEWAY }} \
-               -host=0.0.0.0 -port=80 -portWS=81 -nodeHost=${{ env.L2_RPC_URL_VALIDATOR }} -verbose=true \
-               -logPath=sys_out -dbType=mariaDB -dbConnectionURL="obscurouser:${{ secrets.OBSCURO_GATEWAY_MARIADB_USER_PWD }}@tcp(obscurogateway-mariadb-${{  github.event.inputs.testnet_type }}.uksouth.cloudapp.azure.com:3306)/ogdb" \
-               -rateLimitUserComputeTime=${{ env.GATEWAY_RATE_LIMIT_USER_COMPUTE_TIME }} -rateLimitWindow=${{ env.GATEWAY_RATE_LIMIT_WINDOW }} -maxConcurrentRequestsPerUser=${{ env.GATEWAY_MAX_CONCURRENT_REQUESTS_PER_USER }} '
+              -e OE_SIMULATION=1 \
```

Comment:
we need to be able to switch from simulation ON (for the local container) to simulation OFF ( for testnet)

Reply:
We can do that now. This is github action file which is executed on Azure only.
I guess we will deploy only with non simulation mode, but if you think we need to deploy on Azure also with simulation mode sometimes, then I can introduce new env variable which can control that.

To run it locally we can simply run it like that:
```sh
docker run -d \
  -p 3000:3000 \
  -p 3001:3001 \
  --name local-gateway \
  --ulimit core=-1 \
  -v /tmp/cores:/tmp/cores \
  --security-opt seccomp=unconfined \
  --entrypoint "/home/ten/go-ten/tools/walletextension/main/entry.sh" \
  -e OBSCURO_GATEWAY_VERSION="local" \
  -e OE_SIMULATION=1 \
  gateway \
  ego run /home/ten/go-ten/tools/walletextension/main/main \
  -host=0.0.0.0 -port=3000 -verbose=true
```